sklearn/utils/tests/test_estimator_checks.py

```diff
@@ -620,6 +621,130 @@ def test_check_estimator_pairwise():
     check_estimator(est)
 
 
+class MinimalClassifier:
+
```

Comment:
Don't you need to add:

```python
_estimator_type = "classifier"
```

to get the classifier checks to run on this?

Reply:
True and it is much worse now :)